d3b_dff_cli/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.0.1'
+__version__ = 'v3.0.0'
```

Comment:
```suggestion
__version__ = '3.0.0'
```

This keeps the version number in line with what we've been doing. I didn't notice that I only add the v in the github release tag.